scalajslib/test/src/mill/scalajslib/HelloJSWorldTests.scala

```diff
@@ -50,15 +50,15 @@ object HelloJSWorldTests extends TestSuite {
       )
 
       object `test-utest` extends ScalaJSTests with TestModule.Utest {
-        override def sources = Task.Sources { millSourcePath / "src/utest" }
+        override def sources = Task.Sources { this.millSourcePath / "src/utest" }
```

Comment:
Is the `this.` here necessary?

Reply:
this causes an ambiguity error if it isn't there